templates/default.yaml

```diff
@@ -525,6 +525,11 @@ hostResolver:
 # 🟢 Builtin default: /usr/local
 guestInstallPrefix: null
 
+# When "suspendOnSigInt" is enabled, Lima suspends the VM instead of shutting it down.
+# The VM can be resumed with `limactl start` if the saved state is available.
+# 🟢 Builtin default: false
+suspendOnSigInt: null
```

Comment:
This is not a good name for user interface. We are leaking questionable implementation detail (stopping using SIGINT instead of SIGTERM). The user configuration should not leak such details.

Using suspend for this feature will make it hard to support suspend and resume as used by libvirt:
- https://www.libvirt.org/manpages/virsh.html#suspend
- https://www.libvirt.org/manpages/virsh.html#resume

When a vm is suspended, the qemu process is stopped (I guess with SIGSTOP). When resuming it continues to run (I guess with SIGCONT). Both suspend and resume are very quick. This is a very useful feature for testing, when you want to simulate issues like inaccessible vm.

What vz call suspend is called "save" in libvit:
https://www.libvirt.org/manpages/virsh.html#save

This is much slower, writing and compressing guest memory to disk. This should not be the default behavior since it slow, requires lot of resources, and fragile. For example it will not work if the disks are modified after saving the guest state.

To avoid all the issues, we can call this option `autoSave` or `saveOnStop`. If enabled, instance state is saved when stopping the instance and restored when starting the instance.

Reply:
> To avoid all the issues, we can call this option `autoSave` or `saveOnStop`. If enabled, instance state is saved when stopping the instance and restored when starting the instance.

I prefer `saveOnStop`.

Reply:
I agree that aligning the terminology with libvirt makes sense.